snapcraft/cli/lifecycle.py

```diff
@@ -115,7 +115,9 @@ def init():
     """Initialize a snapcraft project."""
     snapcraft_yaml_path = lifecycle.init()
     echo.info("Created {}.".format(snapcraft_yaml_path))
-    echo.info("Edit the file to your liking or run `snapcraft` to get started")
+    echo.wrapped("Edit the file to your liking or run `snapcraft` to get started. "
+                 "For more information about the format and values accepted in "
+                 "snapcraft.yaml go to https://docs.snapcraft.io/the-snapcraft-format/8337.")
```

Comment:
I'd avoid the ending `.`, as otherwise it will [404](https://docs.snapcraft.io/the-snapcraft-format/8337.) if the url matcher is even a little bit greedy. Turn the structure around if  the lack of punctuation bothers you (i.e. `Go to <url> for more information about <blah>.`)